domains/googse.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "googse-dev",
+    "email": "",
+    "discord": "AngerMinecraftYT#3051"
+  }
+{
```

Comment:
`record` object must be a child object of the main object.
Read the [docs](https://docs.is-a.dev/github_pages/) for more information about setting up GitHub Pages with is-a.dev.